docs/core/whats-new/whats-new-in-dotnet-core-2-0.md

```diff
@@ -0,0 +1,138 @@
+---
+title: What's new in .NET Core 2.0
+description: A description of the new feature found in .NET Core 2.0.
+keywords: .NET, .NET Core
+author: rpetrusha
+ms.author: ronpet
+ms.date: 08/07/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+---
+# What's new in .NET Core 2.0
+
+.NET Core 2.0 includes enhancements and new features in the following areas:
+
+- [Tooling](#tooling)
+- [Language support](#language-support)
+- [Platform improvements](#platform-improvements)
+- [API changes](#api-changes)
+- [Visual Studio integration](#visual-studio-integration)
+- [Documentation improvements](#documentation-improvements) 
+
+## Tooling
+
+**`dotnet restore` runs implicitly**
+
+In previous versions of the .NET Core, you had to run the [dotnet restore](../tools/dotnet-restore.md) command to download dependencies immediately after you created a new project with the [dotnet new](../tools/dotnet-new) command, as well as whenever you added a new dependency to your project. In .NET Core 2.0, `dotnet restore` runs implicitly when the `dotnet new` command executes, as well as if dependencies need to be updated when the `run`, `build`, and `publish` commands execute.
```

Comment:
I'd add that you can turn that off similar to what we've done here: https://github.com/dotnet/announcements/issues/23